### ISSUE TITLE
Disabling Virtual Memory

### ISSUE THREAD

**Issue by ibndias** [riscv/riscv-pk#200]
Hi, I am new to RISCV
is there a way to disable virtual memory address?

Seems like the --disable-vm flag is no longer can be used?

**Comment by aswaterman** [comment]
Right, it's best not to use `pk` at all if you don't want to use virtual memory.

**Comment by ibndias** [comment]
But I need some of its feature, I just want to disable the virtual memory.

Changing the `SATP_MODE_SV32` to `SATP_MODE_OFF` gives me `Store/AMO access fault!`

Could you tell me which are the parts that I should modify in order to disable vm? Because I still have to use `pk`

Thanks!
